javalib/src/main/scala/java/lang/Character.scala

```diff
@@ -1371,189 +1371,433 @@ object Character {
   }
 
   object UnicodeBlock {
-    val SURROGATES_AREA    = new UnicodeBlock("SURROGATES_AREA", 0x0, 0x0)
-    val BASIC_LATIN        = new UnicodeBlock("BASIC_LATIN", 0x0, 0x7f)
-    val LATIN_1_SUPPLEMENT = new UnicodeBlock("LATIN_1_SUPPLEMENT", 0x80, 0xff)
-    val LATIN_EXTENDED_A   = new UnicodeBlock("LATIN_EXTENDED_A", 0x100, 0x17f)
-    val LATIN_EXTENDED_B   = new UnicodeBlock("LATIN_EXTENDED_B", 0x180, 0x24f)
-    val IPA_EXTENSIONS     = new UnicodeBlock("IPA_EXTENSIONS", 0x250, 0x2af)
+    //////////////////
+    // Begin Generated
+    //////////////////
+
+    // scalastyle:off line.size.limit
```

Comment:
Does the scalafmt used in ScalaNative respect the scalastyle:off (and others)
I suspect these are holdovers from the Scala.js port. Perhaps left in baseline
code to reduce churn but probably should not be in new SN code (especially generated).

What does scalafmt do to these large generated tables?  I would expect it to mangle them, but the code in this PR looks OK.